### ISSUE TITLE
Do not recomend to download the script and execute directly on the website

### ISSUE THREAD

**Issue by nonetrix** [xerolinux/xlapit-cli#3]
I assume you don't have bad intentions by doing this, but executing a script with `bash -c "$(curl -fsSL https://tinyurl.com/xtoolkit)"` is a terrible idea. You could easily for example, if the curl user agent is detected change the contents of the bash script being executed to inject malware, or a man in the middle attack could occur, or the server could be taken over just as a example. Instead in my opinion it should tell the user to the following:

1. Download the script with wget
2. Open the script in a text editor and verify it
3. Compare the file with a sha256 listed on the site
4. Then run it with bash 

Also, using a URL shortening service makes this worse because let's say tinyurl is compromised and redirects to a different site entirely

**Comment by DarkXero-dev** [comment]
Thank you for your concern...

But, in an ideal world I would gladly do that simply coz I agree. But we don't live in one. A malicious actor will and can inject bad code in anything not just scripts. If we stopped providing easy ways to do stuff, complicating it by asking them to download and inspect just because these malicious actors then there's no point .. Might as well stop there n give up.. 

Besides I already provided users a link to script so they can inspect it. But asking them to do what you said while not wrong, you think users will bother ? In an ideal world they would but are we in one ? I am a realist.. Reading 2 lines is becoming too hard these days for users to read.. 

Thanks though...
